working-groups/WORKING-GROUPS.md

```diff
@@ -297,33 +300,6 @@ performance/scale/load testing infrastructure
-| Roadmap                    | [Roadmap](https://github.com/orgs/knative/projects/36/views/1)                                                                                              |
-| Forum                      | [knative-dev@](https://groups.google.com/forum/#!forum/knative-dev)                                                                                         |
-| Community Meeting VC       | See the top of the [Meeting notes](https://docs.google.com/document/d/1rpag5-zffHGxAT7V4Nv28C_xx5Ow6L4mZuHbe3ebOQ8/edit)                                   |
-| Community Meeting Calendar | Wednesdays at 9:30am PST <br>[Calendar](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com) |
-| Meeting Notes              | [Notes](https://docs.google.com/document/d/1rpag5-zffHGxAT7V4Nv28C_xx5Ow6L4mZuHbe3ebOQ8/edit)                                                              |
-| Document Folder            | [Folder](https://drive.google.com/drive/folders/1IDDkJ3FD47xFSHY3iA9U2Q8th3Cwdo0K)                                                                          |
-| Repo prefixes              |                                                                                                                                                             |
-| Slack Channel              | [#autoscaling](https://slack.knative.dev/messages/autoscaling)                                                                                              |
-| Github Team WG leads       | [@knative/autoscaling-wg-leads](https://github.com/orgs/knative/teams/autoscaling-wg-leads/members)                                                         |
-
-| &nbsp;                                                         | Leads           | Company | Profile                                             |
-| -------------------------------------------------------------- | --------------- | ------- | --------------------------------------------------- |
-| <img width="30px" src="https://github.com/julz.png">           | Julian Friedman | IBM     | [julz](https://github.com/julz)                     |
-
-| &nbsp;                                                         | Emeritus Leads  | Profile                                             | Duration  |
-| -------------------------------------------------------------- | --------------- | --------------------------------------------------- | --------- |
-| <img width="30px" src="https://github.com/markusthoemmes.png"> | Markus Thömmes  | [markusthoemmes](https://github.com/markusthoemmes) | 2019-2021 |
-| <img width="30px" src="https://github.com/vagababov.png">      | Victor Agababov | [vagababov](https://github.com/vagababov)           | 2019-2021 |
-| <img width="30px" src="https://github.com/josephburnett.png">  | Joseph Burnett  | [josephburnett](https://github.com/josephburnett)   | 2018-2019 |
-
```

Comment:
Move this to "emeritus" or otherwise discontinued, rather than just deleting?

Reply:
I moved it under 'Serving' with designated subgroups.